main/res/values/strings.xml

```diff
@@ -600,6 +600,8 @@
     <string name="init_summary_disabled">Exclude disabled caches</string>
     <string name="init_archived">Exclude Archived</string>
     <string name="init_summary_archived">Exclude archived caches</string>
+    <string name="init_global_wp_extraction_disable">Waypoint Extraction</string>
+    <string name="init_summary_global_wp_extraction_disable">Disable waypoint extraction from personal cache notes</string>
```

Comment:
I suggest something like:
"Globally disable waypoint extraction from personal note. It can still be enabled for individual caches by deactivating the checkbox below the personal note of the cache."

Reply:
> I suggest something like:
> "Globally disable waypoint extraction from personal note. It can still be enabled for individual caches by deactivating the checkbox below the personal note of the cache."

Ok, consider it done. Will go up with the next push